### ISSUE TITLE
confused about  the training input range 

### ISSUE THREAD

**Issue by zxcvads** [megvii-research/PMRID#11]
hi ,i have a question about the input dataset range ,i have see your paper released model input range is(0,256),if i change the input range to(-1,1) or (0,1)，i am confused  about the PSNR change ,can you have a try in different training data range ?

**Comment by bigeagle** [comment]
It's not substantial. Some hyper parameters (such as learning rate) may need to be tuned according to data range.

**Comment by nono0822** [comment]
oh, Thank you for your reply！ I still have a doubt, why do you fix the data range to (0, 256) when training the network, have you tried the training of (0, 1)?

**Comment by nono0822** [comment]
> oh, Thank you for your reply！ I still have a doubt, why do you fix the data range to (0, 256) when training the network, have you tried the training of (0, 1)?

**Comment by bigeagle** [comment]
No specific reason. Some legacy code use (0, 256).
Using (0, 1) is OK and can reach the comparable PSNR.

**Comment by cxy2333-hue** [comment]
> hi ,i have a question about the input dataset range ,i have see your paper released model input range is(0,256),if i change the input range to(-1,1) or (0,1)，i am confused about the PSNR change ,can you have a try in different training data range ?

Excuse me, is the range (0,256) before or after the k-sigma transformation?
